runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -238,10 +256,74 @@ public void addDestination(String logicalGroup, List<String> remoteDestinations)
     }
 
     /**
-     * Remove a destination from the LogReplicationModelMetadataTable for a specific group.
+     * Sets the destinations for a given logical group to the provided list.
+     * <p>
+     * A new logical group will be created if one does not already exist.
+     * Setting destinations for a logical group will overwrite existing destinations.
+     * Can be used to clear existing destinations by passing in an empty list.
+     * Clearing existing destinations will remove the logical group.
+     *
+     * @param logicalGroup String tag that identifies the group of tables to be replicated together.
+     *                     This parameter is case-sensitive.
+     * @param remoteDestinations The clusters which the specified group will be replicated to.
+     *                           Destinations in the list are case-sensitive.
+     * @throws IllegalArgumentException If logicalGroup is null or empty, if remoteDestinations is
+     *                                  null, or if remoteDestinations contains null or empty elements.
+     */
+    public void setDestination(String logicalGroup, List<String> remoteDestinations) {
```

Comment:
should this be `setDestinations` since we are accepting a list?

Reply:
Same as above.